src/client/containers/DataLoader.js

```diff
@@ -28,14 +30,27 @@ const mapDispatchToProps = dispatch => {
 
 function maybeLoadData(props) {
```

Comment:
This is not a great name. Use something more explicit like "loadDataUsingCache" or "returnCachedOrLoad"